doc/self-hosting.md

```diff
@@ -139,8 +139,17 @@ If you want to host the frontend on a separate server, such as cloud storage
 or a CDN, you can do that. Just copy the built `index.html` there.
 
 To help out users, you can make the Shields server redirect the server root.
-Set the `FRONTEND_REDIRECT_URL` environment variable:
+Set the `REDIRECT_URI` environment variable:
 
 ```sh
-FRONTEND_REDIRECT_URL=http://my-custom-shields.s3.amazonaws.com/
+REDIRECT_URI=http://my-custom-shields.s3.amazonaws.com/
 ```
+
+If you want to use server suggestions, you should also set `ALLOWED_ORIGIN`:
```

Comment:
In line 20 in this file, there is an info that one should build frontend using `BASE_URL=https://your-server.example.com npm run build:production`
`build:production` runs `make footer-production-transform`, which sets `img.shields.io` as a suggestions server (`var origin = 'https://img.shields.io';` in `index.html`). Requests from locally build frontend will go to img.shields.io.

Reply:
Thanks for noticing that. It should be fixed as part of #1273, which gets rid of the makefile transform and ships the suggestions server through the bundle.